### DISCUSSION TITLE
买了2年的专业版本，试用2天，网络很不稳定，经常连接不上， 请退款

### DISCUSSION THREAD

**Discussion by Nyainqentanglha** [getlantern/lantern-forum-cn#908] [General]
买了2年的专业版本，试用2天，网络很不稳定，经常连接不上， 请退款！
我的邮箱：307328001@qq.com

**Comment by MaxMKMax** [comment]
@Nyainqentanglha 非常抱歉影响了您的体验感. 请您提供我支付单据. 

您要不要再试看多几天, 先别退款?
